### ISSUE TITLE
MAPS no color defined for incidence values between 5 and 15

### ISSUE THREAD

**Issue by Rubber1Duck** [marlon360/rki-covid-api#80]
MAPS no color defined for incidence values between 5 and 15.
See 
rki-covid-api/src/configuration/colors.ts 

```
export const weekIncidenceColorRanges = {
    ranges: [
        {
            min: 0,
            max: 1,
            color: "#2D81B8"
        },
        {
            min: 1,
            max: 5,
            color: "#7FD38D"
        },
        {
            min: 15,
            max: 25,
            color: "#FEFFB1"
        }, .............. 
```
Solution: insert a color for incidence values from 5 to 15, or extend the range from 2nd color to max:15

one more Question, what happens if a incidence value is exact the value of the border?
eg. Incidence=1 , witch collor will be chosen 1st one or 2nd one
is it not the better way to do it like this (maybe this is catched by the code)

```
export const weekIncidenceColorRanges = {
    ranges: [
        {
            min: 0,
            max: <1,
            color: "#2D81B8"
        },
        {
            min: >=1,
            max: <5,
            color: "#7FD38D"

and so on 
       
```

**Comment by Rubber1Duck** [comment]
> one more Question, what happens if a incidence value is exact the value of the border?
eg. Incidence=1 , witch collor will be chosen 1st one or 2nd one

is it not the better way to do it like this (maybe this is catched by the code)

export const weekIncidenceColorRanges = {
    ranges: [
        {
            min: 0,
            max: <1,
            color: "#2D81B8"
        },
        {
            min: >=1,
            max: <5,
            color: "#7FD38D"

and so on 

I have yust seen that this is catched by the code!

**Comment by Rubber1Duck** [comment]
> 
> one more Question, what happens if a incidence value is exact the value of the border?
> eg. Incidence=1 , witch collor will be chosen 1st one or 2nd one
> is it not the better way to do it like this (maybe this is catched by the code)
> 
> ```
> export const weekIncidenceColorRanges = {
>     ranges: [
>         {
>             min: 0,
>             max: <1,
>             color: "#2D81B8"
>         },
>         {
>             min: >=1,
>             max: <5,
>             color: "#7FD38D"
> 
> and so on 
>        
> ```

i have yust seen that this is catched by the code!

**Comment by marlon360** [comment]
Thanks again @Rubber1Duck 

Fixed with e0161a2db6031a8e174c3d567422e14629536838
